frontend/app/.eslintrc.js

```diff
@@ -17,7 +17,9 @@ module.exports = {
           '/backend_mappings.*/',
           '/notification_messages.missing_api_key.*/',
           '/premium_components.*/',
-          '/transactions.query_status.*/'
+          '/transactions.query_status.*/',
+          '/transactions.query_status_events.*/',
+          '/transactions.events.headers.*/'
```

Comment:
We need to figure out something for this, just ignoring everything would make this hard to maintain.